src/main/kotlin/racingcar/model/Car.kt

```diff
@@ -0,0 +1,14 @@
+package racingcar.model
+
+import camp.nextstep.edu.missionutils.Randoms
+
+class Car(val name: String) {
+    var position = 0
```

Comment:
position이 var로 공개되어 있어서 외부(Race)에서 직접 position++을 수행할 수 있네요.
이렇게 되면 Car의 내부 상태를 외부에서 직접 변경할 수 있어, 조금 위험할 것 같아요.
사실 저도 이렇게 코드를 작성했는데 지금 생각났습니다...ㅎㅎㅎ

```suggestion
    private var position = 0
    fun getPosition() = position
```